flink-table-store-common/src/test/java/org/apache/flink/table/store/format/FileStatsExtractorTestBase.java

```diff
@@ -128,6 +129,9 @@ private Object createField(LogicalType type) {
                         randomString(random.nextInt(varCharType.getLength()) + 1));
             case BOOLEAN:
                 return random.nextBoolean();
+            case BINARY:
```

Comment:
I think we should add a case in `FileStoreTableTestBase` to test read and write.

Reply:
@JingsongLi, the `testReadFilter` in `FileStoreTableTestBase` could cover the test case to test read and write. PTAL.

Reply:
We can add test for new types.